target/linux/ath79/image/nand.mk

```diff
@@ -150,6 +150,23 @@ define Device/glinet_gl-e750
 endef
 TARGET_DEVICES += glinet_gl-e750
 
+define Device/glinet_gl-xe300-nor-nand
+  SOC := qca9531
+  DEVICE_VENDOR := GL.iNet
+  DEVICE_MODEL := GL-XE300
+  DEVICE_PACKAGES := kmod-ath10k-ct ath10k-firmware-qca9887-ct kmod-usb2
+  SUPPORTED_DEVICES += glinet,gl-xe300-nor-nand
```

Comment:
This line can be dropped.

Reply:
Isn't that mandatory when performing OpenWrt sysupgrade?

sysupgrade tells me that the firmware is only compatible for 'glinet,gl-xe300-nor-nand' devices and I'm forced to use the '-F' flag to perform sysupgrade.

Reply:
This value is equal to the global default, and thus redundant:
https://github.com/openwrt/openwrt/blob/master/include/image.mk#L407

Reply:
Done, thanks.